ERROR - 2022-02-08 21:22:26 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-02-08 21:22:26 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\Xampp\htdocs\Ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Xampp\htdocs\Ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\Xampp\htdocs\Ci4\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Xampp\htdocs\Ci4\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\Xampp\htdocs\Ci4\app\Controllers\Signincontroller.php(26): CodeIgniter\BaseModel->first()
#5 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(825): App\Controllers\SigninController->loginAuth()
#6 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#7 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\Xampp\htdocs\Ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-02-08 21:26:33 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-02-08 21:26:33 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\Xampp\htdocs\Ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Xampp\htdocs\Ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\Xampp\htdocs\Ci4\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Xampp\htdocs\Ci4\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\Xampp\htdocs\Ci4\app\Controllers\Signincontroller.php(26): CodeIgniter\BaseModel->first()
#5 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(825): App\Controllers\SigninController->loginAuth()
#6 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#7 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\Xampp\htdocs\Ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-02-08 21:41:47 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-02-08 21:41:47 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\Xampp\htdocs\Ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Xampp\htdocs\Ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\Xampp\htdocs\Ci4\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Xampp\htdocs\Ci4\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\Xampp\htdocs\Ci4\app\Controllers\Signincontroller.php(26): CodeIgniter\BaseModel->first()
#5 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(825): App\Controllers\SigninController->loginAuth()
#6 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#7 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\Xampp\htdocs\Ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-02-08 21:43:43 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-02-08 21:43:43 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\Xampp\htdocs\Ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Xampp\htdocs\Ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\Xampp\htdocs\Ci4\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Xampp\htdocs\Ci4\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\Xampp\htdocs\Ci4\app\Controllers\Signincontroller.php(26): CodeIgniter\BaseModel->first()
#5 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(825): App\Controllers\SigninController->loginAuth()
#6 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#7 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\Xampp\htdocs\Ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
ERROR - 2022-02-08 21:48:26 --> Error connecting to the database: Access denied for user ''@'localhost' (using password: NO)
CRITICAL - 2022-02-08 21:48:26 --> Unable to connect to the database.
Main connection [MySQLi]: Access denied for user ''@'localhost' (using password: NO)
#0 C:\Xampp\htdocs\Ci4\system\Database\BaseConnection.php(577): CodeIgniter\Database\BaseConnection->initialize()
#1 C:\Xampp\htdocs\Ci4\system\Database\BaseBuilder.php(1455): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#2 C:\Xampp\htdocs\Ci4\system\Model.php(210): CodeIgniter\Database\BaseBuilder->get()
#3 C:\Xampp\htdocs\Ci4\system\BaseModel.php(622): CodeIgniter\Model->doFirst()
#4 C:\Xampp\htdocs\Ci4\app\Controllers\Signincontroller.php(26): CodeIgniter\BaseModel->first()
#5 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(825): App\Controllers\SigninController->loginAuth()
#6 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\SigninController))
#7 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 C:\Xampp\htdocs\Ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-02-08 23:45:17 --> Invalid file: edit_user.php
#0 C:\Xampp\htdocs\Ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('edit_user.php')
#1 C:\Xampp\htdocs\Ci4\system\Common.php(1089): CodeIgniter\View\View->render('edit_user', Array, true)
#2 C:\Xampp\htdocs\Ci4\app\Controllers\UserCrud.php(35): view('edit_user', Array)
#3 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(825): App\Controllers\UserCrud->singleUser('1')
#4 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserCrud))
#5 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\Xampp\htdocs\Ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
CRITICAL - 2022-02-08 23:57:04 --> Invalid file: edit_user.php
#0 C:\Xampp\htdocs\Ci4\system\View\View.php(198): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('edit_user.php')
#1 C:\Xampp\htdocs\Ci4\system\Common.php(1089): CodeIgniter\View\View->render('edit_user', Array, true)
#2 C:\Xampp\htdocs\Ci4\app\Controllers\UserCrud.php(35): view('edit_user', Array)
#3 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(825): App\Controllers\UserCrud->singleUser('1')
#4 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(412): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\UserCrud))
#5 C:\Xampp\htdocs\Ci4\system\CodeIgniter.php(320): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#6 C:\Xampp\htdocs\Ci4\public\index.php(37): CodeIgniter\CodeIgniter->run()
#7 {main}
